packages/safe-ds-lang/src/language/safe-ds-module.ts

```diff
@@ -150,6 +154,9 @@ export const SafeDsModule: Module<SafeDsServices, PartialLangiumServices & SafeD
         PackageManager: (services) => new SafeDsPackageManager(services),
         SettingsProvider: (services) => new SafeDsSettingsProvider(services),
     },
+    runtime: {
```

Comment:
I suppose the optional parameter should be passed somewhere here?
https://github.com/Safe-DS/DSL/blob/21298d0a3be7c9e0994614641471cb0344826696/packages/safe-ds-lang/src/language/safe-ds-module.ts#L209